src/python/WMCore/Database/DBFormatter.py

```diff
@@ -109,7 +140,7 @@ def formatCursor(self, cursor, size=10):
         Use fetchmany(size = default arraysize = 50)
 
         """
-	if type(cursor.keys) == types.MethodType:
+        if type(cursor.keys) == types.MethodType:
```

Comment:
Yes, and I had no intention to fix all bugs. But this specific one is captured
by pylint.

On  0, Eric Vaandering <notifications@github.com> wrote:
> > @@ -109,7 +140,7 @@ def formatCursor(self, cursor, size=10):
> >          Use fetchmany(size = default arraysize = 50)
> >  
> >          """
> > -	if type(cursor.keys) == types.MethodType:
> > +        if type(cursor.keys) == types.MethodType:
> 
> This appears to be a genuine bug
> 
> ---
> Reply to this email directly or view it on GitHub:
> https://github.com/dmwm/WMCore/pull/6260/files#r39919917